### PR TITLE
[CM-699] Web like rating view

### PR DESCRIPTION
**Pull request by shilwantk**
## Summary
Added feedback icons to show rating labels

### CODE REVIEW

File: Sources/Views/ALKInformationCell.swift [hunk 1]
```diff
@@ -76,8 +76,35 @@ final class ALKInformationCell: UITableViewCell {
 
     func update(viewModel: ALKMessageViewModel) {
         self.viewModel = viewModel
+        
+        //adding rating icons to the label
+        var ratingImage = UIImage()
+        guard let feedbackString =  viewModel.metadata? ["feedback"] as? String else { return }
+        do {
+            let feedbackDictionary = try? JSONSerialization.jsonObject(with: feedbackString.data(using: .utf8)!, options: []) as? [String:Int]
+            switch feedbackDictionary!["rating"]! {
```
Comment (mukeshthawani) [inline Sources/Views/ALKInformationCell.swift:85]: Add a check for this values.

File: Sources/Views/ALKInformationCell.swift [hunk 2]
```diff
@@ -76,8 +76,35 @@ final class ALKInformationCell: UITableViewCell {
...
+            case 1:
+                ratingImage = UIImage(named: "sadEmoji", in: Bundle.applozic, compatibleWith: nil)!
+            case 5:
+                ratingImage = UIImage(named: "confusedEmoji", in: Bundle.applozic, compatibleWith: nil)!
+            case 10:
+                ratingImage = UIImage(named: "happyEmoji", in: Bundle.applozic, compatibleWith: nil)!
+            default:
+                print("incoreect data")
+            }
```
Comment (mukeshthawani) [inline Sources/Views/ALKInformationCell.swift:94]: Create a separate function to get the rating image based on the value. I guess we already have it, so check once.

File: Sources/Views/ALKInformationCell.swift [hunk 3]
```diff
@@ -76,8 +76,35 @@ final class ALKInformationCell: UITableViewCell {
...
+           print(error)
+        }
 
-        messageView.text = viewModel.message
+        let imageAttachment = NSTextAttachment()
+        imageAttachment.image = ratingImage
+        guard let attachedImage = imageAttachment.image else { return }
+        imageAttachment.bounds = CGRect(x: 0, y: -5 , width: attachedImage.size.width, height: attachedImage.size.height)
+        let imageString = NSAttributedString(attachment: imageAttachment)
+        var textString = NSMutableAttributedString(string: "User " + viewModel.message! + "  ")
```
Comment (mukeshthawani) [inline Sources/Views/ALKInformationCell.swift:105]: Move `User` String to localized strings.

**Comment by mukeshthawani** [comment]
Also, add a Changelog entry.

File: Sources/Views/ALKInformationCell.swift (hunk 2 above)
Comment (shilwantk) [reply Sources/Views/ALKInformationCell.swift:95]: We do not have the function in Applozic

Comment (mukeshthawani) [reply Sources/Views/ALKInformationCell.swift:95]: Then just do this:
> Create a separate function to get the rating image based on the value.

**Review by mukeshthawani — APPROVED** [review]

**Pull request closed without merging**